Library/Koala/Koala.swift

```diff
@@ -1591,6 +1591,20 @@ public final class Koala {
     self.track(event: "Changed Live Stream Orientation", properties: props)
   }
 
+  public func trackClosedLiveStream(project: Project,
```

Comment:
since this is a brand new event and we can decide how it's done, let's make it a single event with a `type` property that determines countdown/live/replay.

also did you wanna try for the stretch goal of tracking a property for the total amount of time spent in the controller?